encryption-at-rest.md

```diff
@@ -109,6 +109,17 @@ To monitor encryption at rest, if you deploy TiKV with Grafana, you can look at
 
 For debugging, the `tikv-ctl` command can be used to dump encryption metadata such as encryption method and data key id used to encryption the file, as well as list of data keys. Since the operation can expose sensitive data, it is not recommended to use in production. Please refer to [TiKV Control](/tikv-control.md#dump-encryption-metadata] document.
 
+### Compatibility with earlier TiKV versions
+
+An optimization is introduced in TiKV 4.0.9 to reduce IO and mutex contention overhead to manage encryption metadata. The optimization is gated behind `security.encryption.enable-file-dictionary-log` config. The config defaults to on for TiKV 4.0.9 and later versions. When the config is turned on, the data format of encryption metadata is unrecognizable by TiKV v4.0.8 or ealier. If the config is turned on (e.g. using TiKV v4.0.9 or later with encryption-at-rest and default `enable-file-dictionary-log` config), then downgrade to TiKV v4.0.8 or earlier, TiKV will fail to start, with an error in the info log similar to the following one:
```

Comment:
+ We need to spell out `config` as `configuration` in technical documentation.
+ `turn on` --> `enable`
+ Could you please add this configuration in [TiKV Configuration File](https://github.com/pingcap/docs/blob/master/tikv-configuration-file.md), put most of its parameter information there, and reference to that file in this passage? That's what we normally do in TiDB tech docs.
```suggestion
To reduce the overhead caused by I/O and mutex contention when you manage the encryption metadata, an optimization is introduced in TiKV v4.0.9 and controlled by `security.encryption.enable-file-dictionary-log` in the TiKV configuration file. This configuration parameter takes effect only on TiKV v4.0.9 or later versions. When it is enabled (by default), the data format of encryption metadata is unrecognizable by TiKV v4.0.8 or earlier versions.

For example, assume that you use TiKV v4.0.9 or later with encryption-at-rest and the default `enable-file-dictionary-log` configuration. If you downgrade your cluster to TiKV v4.0.8 or earlier, TiKV will fail to start, with an error in the info log similar to the following one:
```

Reply:
"when you manage the encryption metadata" is not right. It is TiKV, not "you"(the user) managing the metadata.

Reply:
updated.